docs/core/diagnostics/cli-tools/dotnet-trace.md

```diff
@@ -0,0 +1,182 @@
+---
+title: dotnet-trace - .NET Core
+description: Installing and using the dotnet-trace command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# Trace for performance analysis utility (`dotnet-trace`)
+
+The `dotnet-trace` tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It's built around the `EventPipe` technology of the .NET Core runtime as a cross-platform alternative to ETW on Windows and LTTng on Linux, which only work on a single platform. With `EventPipe`/`dotnet-trace`, we're trying to deliver the same experience on Windows, Linux, or macOS.
+
+> [!NOTE]
+> `dotnet-trace` can be used on any .NET Core applications using versions .NET Core 3.0 Preview 7 or later.
+
+## Installing `dotnet-trace`
+
+To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace):
+
+```
+dotnet tool install --global dotnet-trace
+```
+
+For details and other options, see [Installing the diagnostics tools](installing.md).
+
+## Using `dotnet-trace`
+
+To collect traces, using `dotnet-trace`, you'll need to:
+
+- First, find out the process identifier (pid) of the .NET Core application to collect traces from.
+
+  - On Windows, there are options such as using the task manager or the `tasklist` command on the cmd window.
+  - On Linux, the trivial option could be using `pidof` on the terminal window.
+
+You may also use the command `dotnet-trace list-processes` command to find out what .NET Core processes are running, along with their process IDs.
+
+- Then, run the following command:
+
+```bash
+dotnet-trace collect --process-id <PID> --providers Microsoft-Windows-DotNETRuntime
```

Comment:
I think we could start with no arguments other than process-id which should give a default experience of collecting thread time sampling information + various other useful runtime information (GC, Jit, assembly loading)